yt/visualization/volume_rendering/input_events.py

```diff
@@ -1,13 +1,20 @@
 # This is a part of the experimental Interactive Data Visualization
-import random
+import os
+import time
 from collections import defaultdict, namedtuple
 
-import cyglfw3 as glfw
 import numpy as np
-from matplotlib import cm as cm
-from OpenGL import GL as GL
+import OpenGL.GL as GL
```

Comment:
Is there a reason to write `import OpenGL.GL as GL` instead of `from OpenGL import GL`?

Reply:
Hmm, I don't know.  I can't remember why I did this.